locales/locales.json

```diff
@@ -1692,8 +1692,7 @@
       "noPermission": "Standortzugriff erforderlich",
       "loading": "Bestimme Standort...",
       "notAvailable": "Standortbestimmung deaktiviert",
-      "timeout": "Standortbestimmung fehlgeschlagen",
-      "refresh": "Liste der Orte aktualisieren"
```

Comment:
Why has this been removed? Is this not needed anymore?

Reply:
The locales branch was behind its master. Fixed it.